integration_tests/integration_test.py

```diff
@@ -226,14 +233,19 @@ def test_demultiplexing(self):
         assert self._test_success
 
     def test_demultiplexing_aborted(self):
-        self.setup_test('sample', 'test_demultiplexing', 'demultiplexing')
-        with patch('analysis_driver.pipelines.demultiplexing.Setup._run', side_effect=SequencingRunError):
+        self.setup_test('sample', 'test_demultiplexing_aborted', 'demultiplexing')
+
+        mocked_clarity_run = MockedRunProcess(udf={'Run Status': 'RunAborted'}, container=mocked_flowcell_pooling)
+        with patch('egcg_core.clarity.get_run', return_value=mocked_clarity_run):
             exit_status = client.main(['--run'])
-            self.assertEqual('exit status', exit_status, 0)
 
-            self.expect_equal(
-                rest_communication.get_document('analysis_driver_procs')['status'], 'aborted'
-            )
+        self.assertEqual('exit status', exit_status, 0)
+        ad_proc = rest_communication.get_document('analysis_driver_procs', where={'dataset_name': self.run_id})
+
+        self.expect_equal(
+            ad_proc['status'], 'aborted', 'pipeline status'
+        )
+        self.expect_stage_data([('setup', 9)], analysis_driver_proc=ad_proc['proc_id'])
```

Comment:
This keyword arg won't work - should be `where=...`

Reply:
This is fixed now. Tests did pass cause there is only one stage at this point